.php_cs.dist

```diff
@@ -15,13 +15,16 @@ HEADER;
 return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
+        '@PSR2' => true,
```

Comment:
`@PSR2` is already included in `@Symfony` (cf. https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.16/src/RuleSet.php#L59-L60)

Reply:
Fixed